cmake/std/atdm/cee-rhel6/environment.sh

```diff
@@ -110,6 +110,9 @@ elif [[ "$ATDM_CONFIG_COMPILER" == "GNU-4.9.3_OPENMPI-1.10.2" ]] ; then
   export ATDM_CONFIG_SUPERLUDIST_LIBS=${SUPERLUDIST_ROOT}/lib/libsuperlu_dist_4.2.a
 elif [ "$ATDM_CONFIG_COMPILER" == "INTEL-18.0.2_MPICH2-3.2" ]; then
   module load sparc-dev/intel-18.0.2_mpich2-3.2
+  export OMP_NUM_THREADS=2
```

Comment:
@rppawlo, see, here is `OMP_NUM_THREADS=2` so this is testing real threaded code as well as any of the automated OpenMP builds being run in any of the builds submitting to CDash.  (Again, doing better requires extending ctest as described in #2422.)